pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -362,7 +362,11 @@ func (r *NetworkInfoReconciler) setupWithManager(mgr ctrl.Manager) error {
 
 // Start setup manager and launch GC
 func (r *NetworkInfoReconciler) Start(mgr ctrl.Manager) error {
-	err := r.setupWithManager(mgr)
+	err := r.RegisterAllNetworkInfo(context.Background())
```

Comment:
This change may not work for the standby Pod with the HA mode. It calls `r.RegisterAllNetworkInfo` before r.setupWithManager(mgr), it means loading all VPCNetworkConfigurations before manager.Start. It may have issues in HA mode on the standby Pod. For example, at time1, the pod loads all VPC Network configurations from kube apiserver, then it stands by and waits for itself to be the master. During this time, more VPC Network Configurations are created, but they are not existing in the cache. After the Pod is elected as the master, the Pod actually does not have the new resources, it still needs the Reconcile to sync the resources?

Reply:
the k8s client in manager struct is not available before setupWithManager, will cause nil memory error

Reply:
slave will be blocked before start controller.